### PR TITLE
!!![TASK] Remove old rendering

### PR DESCRIPTION
**Pull request by jaapio**
This PR will remove the older sphinx rendering from this action. 

Do not merge before the deadline to migrate to the new rendering

### CODE REVIEW

**Comment by linawolf** [comment]
Setting this to draft so it does not accidently get rendered

**Comment by jaapio** [comment]
Question is... Should we automate that? It might be better to just break as we might break the docs?

**Comment by linawolf** [comment]
We promised to make an on-the-fly migration

**Comment by garvinhicking** [comment]
From my experience, while the guides.xml automigration works quite flawless, usually there's always at least 1 or 2 directives that do not work out (shortcuts in Includes.txt, image inclusion). So probably the auto migration will never create a document that renders without errors (thus - will never get deployed).

But if there's a chance that 10 or 15% would work with an automigraiton, we should do that, it's better than 0% and letting them all fail.

**Comment by linawolf** [comment]
These directives usually cause warnings not errors. So it would get rendered but a few links or directives would not work. So it would get rendered but not Flawlessly as if you had done a manual migration and ensured there are no warnings

**Comment by jaapio** [comment]
Ok, I will re-add the step to detect the guides.xml and create one if it doesn't exist using the migration command.

**Comment by jaapio** [comment]
I did add a migration step

File: action.yml [hunk 1]
```diff
@@ -42,11 +42,28 @@ runs:
...
       working-directory: t3docsproject
       shell: bash
       run: mkdir -p RenderedDocumentation/Result/project/0.0.0
 
+    - id: migrate
+      if: steps.enable_guides.outputs.MIGRATED == 'false'
+      shell: bash
+      working-directory: t3docsproject
+      run: |
+        echo "::warning file=Documentation/Settings.cfg,line=1,title=Please migrate::Your project not yet using the php based rendering, please migrate: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Howto/Migration/Index.html"
```
Comment (garvinhicking) [inline action.yml:60]:
```suggestion
        echo "::warning file=Documentation/Settings.cfg,line=1,title=Please migrate::Forced migration activated. Your project is not yet using the PHP-based rendering, please properl migrate: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Howto/Migration/Index.html"
```

**Comment by jaapio** [comment]
Yes, we need to test this. But as far as I know there is no way to do that without a merge.

**Comment by fe-hicking** [comment]
Yes, of course - I meant when it's ready we should find a timeslot to merge it, run a test rendering (on one non migrated documentation and one "proper") and see if it works, and be able to rollback or improve quickly. 
Let's coordinate on slack? Maybe it's sufficient for us two to coordinate, but you can also do that with Lina, or the three of us.

**Comment by linawolf** [comment]
I am very much for a test run

**Pull request merged**